cdhweb/static_src/global/components/event-hero.scss

```diff
@@ -0,0 +1,115 @@
+/* 
+Event hero. Similar to standard hero. Key differences:
+- image ratio (TODO – maybe?)
```

Comment:
I feel like all the hero aspect ratios are a bit weird and I will go through this as a design QA once they're all built.